menu_launcher.py

```diff
@@ -650,15 +228,15 @@ def run_plugins(path_dirs, action):
             plugins[plug] = config.get("plugins", plug)
 
         # get number of installed cores
-        cores = len(get_installed_cores(path_dirs))
+        cores = len(ast.literal_eval(check_output("python2.7 "+path_dirs.info_dir+'get_status.py cores', shell=True)))
```

Comment:
`literal_eval` should always be in a try/catch in case it returns an unexpected object